conbench/entities/run.py

```diff
@@ -249,6 +247,30 @@ def get_default_baseline_run(
 
         return Run.get(closest_run_id)
 
+    def get_candidate_baseline_runs(self) -> Dict[str, Optional[_FoundBaselineRun]]:
+        """Return information about a few different candidate baseline runs, including
+        on the parent commit, fork-point commit, and head-of-default-branch commit.
+
+        See docstring of _get_baseline_run_from_baseline_commit() for how these are
+        found.
+        """
+        return {
+            "parent": self._get_baseline_run_from_baseline_commit(
+                self.commit.get_parent_commit()
```

Comment:
This is always _the_ direct, single parent in the commit graph, right?

Reply:
Yes!